### PR TITLE
History page

### PR DESCRIPTION
**Pull request by p-szalast**

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -50,20 +50,104 @@
...
         }
       ],
       "address": {
-        "firstName": "test name 2",
-        "lastName": "",
+        "firstName": "Przemysław",
+        "lastName": "Szalast",
         "phoneNumber": "",
-        "street": "",
+        "street": "Topolowa 8",
```
Comment (cyborgx99) [inline db.json:144]: Can we visit? 😃

File: src/pages/CartPage/CartPage.tsx [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-import Cart from "../../components/Cart/CartList";
+import CartList from "../../components/Cart/CartList";
 
 const CartPage = () => {
   return (
-    <StyledPage>
-      <Hero />
-      <Cart />
-    </StyledPage>
+    <div style={{ height: "100vh" }}>
```
Comment (cyborgx99) [inline src/pages/CartPage/CartPage.tsx:8]: It's better to not use inline styles. It's usually for "a quick test"

**Review by cyborgx99 — APPROVED** [review]

**Review by StepinaHelen — APPROVED** [review]

File: db.json (hunk 1 above)
Comment (StepinaHelen) [reply db.json:144]: > Can we visit? 😃

we must))

File: src/pages/CartPage/CartPage.tsx (hunk 2 above)
Comment (p-szalast) [reply src/pages/CartPage/CartPage.tsx:8]: This is what Helen wrote to test yesterday :)

**Pull request merged**